backend/src/sync.js

```diff
@@ -344,5 +399,31 @@ async function syncMissingNearcoreState() {
   );
 }
 
+async function syncGenesisState() {
+  const genesisConfig = await nearRpc.sendJsonRpc(
+    "EXPERIMENTAL_genesis_config"
+  );
+  const genesisTime = moment(genesisConfig.genesis_time).valueOf();
+  const limit = 100;
+  let offset = 0,
+    batchCount;
+  do {
+    let pagination = { offset, limit };
+    console.log(`Sync Genesis records from ${offset} to ${offset + limit}`);
+    const genesisRecords = await nearRpc.sendJsonRpc(
+      "EXPERIMENTAL_genesis_records",
+      [pagination]
+    );
```

Comment:
Considering that it has a good chance to fail at least once, there is very little chance that this sync will ever succeed since it will always start from the very beginning and will most probably fail again.